content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,255 @@
+
+# Verify existing Kubernetes cluster
+
+After the cluster is created following the [Kubernetes Cluster as Code - Part 1 blog post](https://developer.hpe.com/blog/kubernetes-clusters-as-code-part1/), launch HPE GreenLake Central console and verify that the cluster is present under the appropriate tenant.
+
+You should see the **tf-test** cluster present under **Dashboard -> Manage your Private Cloud -> Containers**.
+
+![](/img/cluster_list.jpg)
+
+Below is the reference Terraform configuration file for the existing cluster.
+
+```hcl
+terraform {
+  required_providers {
+    hpegl = {
+      source  = "hpe/hpegl"
+      version = ">= 0.2.2"
+    }
+  }
+}
+ 
+provider hpegl {
+  caas {
+  }
+}
+ 
+variable "HPEGL_SPACE" {
+  type = string
+}
+ 
+data "hpegl_caas_site" "blr" {
+  name = "BLR"
+  space_id = var.HPEGL_SPACE
+ }
+ 
+data "hpegl_caas_cluster_blueprint" "bp" {
+  name = "demo"
+  site_id = data.hpegl_caas_site.blr.id
+}
+ 
+resource hpegl_caas_cluster test {
+  name         = "tf-test"
+  blueprint_id = data.hpegl_caas_cluster_blueprint.bp.id
+  site_id      = data.hpegl_caas_site.blr.id
+  space_id     = var.HPEGL_SPACE
+}
+```
+
+# Add worker node to Kubernetes cluster resource
+
+You can scale the cluster by adding a worker node. The following **worker_nodes** attributes are specified to add or modify node pools in the declared Kubernetes cluster resource.
+
+1. **name**: Fill in the name which would ideally represent each node pool.
+2. **machine_blueprint_id**: Fill in the ID for the machine blueprint which is already present in HPE GreenLake Central for your tenant. We use the machine blueprint data source to retrieve the machine blueprint ID.
+3. **count**: Add the number of nodes to be present as part of this node pool.
+
+Below is the reference Terraform configuration for creating the cluster with additional nodes.
+
+```hcl
+terraform {
+  required_providers {
+    hpegl = {
+      source  = "hpe/hpegl"
+      version = ">= 0.2.2"
+    }
+  }
+}
+ 
+provider hpegl {
+  caas {
+  }
+}
+ 
+variable "HPEGL_SPACE" {
+  type = string
+}
+ 
+data "hpegl_caas_site" "blr" {
+  name = "BLR"
+  space_id = var.HPEGL_SPACE
+ }
+ 
+data "hpegl_caas_cluster_blueprint" "bp" {
+  name = "demo"
+  site_id = data.hpegl_caas_site.blr.id
+}
+ 
+data "hpegl_caas_machine_blueprint" "standard_worker" {
+  name = "standard-worker"
+  site_id = "data.hpegl_caas_site.blr.id"
+}
+
+resource hpegl_caas_cluster test {
+  name         = "tf-test"
+  blueprint_id = data.hpegl_caas_cluster_blueprint.bp.id
+  site_id      = data.hpegl_caas_site.blr.id
+  space_id     = var.HPEGL_SPACE
+
+  worker_nodes {
+    name = "test-node-pool"
+    machine_blueprint_id = data.hpegl_caas_machine_blueprint.standard_worker.id
+    count = "1"
+  }
+}
+```
+
+### *Trivia*
```

Comment:
Not sure what you wanted to mean here. 
Is it to provide a "note" to the readers about blueprint?
If it is the case, maybe set this paragraph as a Note:

>***Note: Machine blueprint are used ....***